frontend/app/professor/test-simulations/page.tsx

```
@@ -1417,309 +1817,603 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
-                    Linear Flow Integration Active
+      
+      <div className="flex-1 ml-20 flex flex-col">
+        {/* Top Navigation Bar */}
+        <div className="bg-white px-6 py-4">
+          <div className="flex items-center justify-between">
+            <div className="flex items-center gap-4">
+              <button
+                onClick={() => router.back()}
+                className="text-gray-600 hover:text-gray-900"
+              >
+                <ArrowLeft className="w-5 h-5" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)